modules/publicize/publicize-jetpack.php

```diff
@@ -452,7 +452,10 @@ function test_connection( $service_name, $connection ) {
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
 	 */
-	function save_publicized( $post_ID, $post, $update ) {
+	function save_publicized( $post_ID, $post = null, $update = null ) {
+		if ( is_null( $post ) ) {
```

Comment:
Just wondering why you check for a numeric post id here https://github.com/Automattic/jetpack/pull/6051/files#diff-b42b8628b3d5d856de376e02baace310R254 but not here?